schema

```diff
@@ -0,0 +1,33 @@
+collection users
+{
+	name : "mayuri",
+	surname: "ardad",
+	login_credentials:{
+						username: "mayuri"
+						password: "something"
+
+						},
+	role: {
+			position: "admin",
+			people_assigned: [....],
+			},
+	posts: [.....]
+}
+
+
+
+collection posts
+
+{
+	text: "hello",
+	image: ObjectId('....'),
```

Comment:
timestamp missing in posts eg: created_at, updated_at, modified_at, published_at, etc...